src/tests/test_meeting_api.py

```diff
@@ -9,28 +12,53 @@
 
 import json
 
+example_meeting = {
```

Comment:
how about adding this to the setup method of the following class. So that it would be assigned "as is"  in each test. I saw the same thing in other tests too.